android/java/org/chromium/chrome/browser/BraveRewardsNativeWorker.java

```diff
@@ -587,11 +587,10 @@ private void setNativePtr(long nativePtr) {
 
     @CalledByNative
     public void OnPublisherInfo(int tabId) {
-        @PublisherStatus int pubStatus = GetPublisherStatus(tabId);
-        boolean verified = (pubStatus == BraveRewardsPublisher.CONNECTED
-                                   || pubStatus == BraveRewardsPublisher.UPHOLD_VERIFIED
-                                   || pubStatus == BraveRewardsPublisher.BITFLYER_VERIFIED
-                                   || pubStatus == BraveRewardsPublisher.GEMINI_VERIFIED)
+        int pubStatus = GetPublisherStatus(tabId);
+        boolean verified = (pubStatus == PublisherStatus.UPHOLD_VERIFIED
```

Comment:
For short: `boolean verified = pubStatus != PublisherStatus.NOT_VERIFIED;`. 🙂

Reply:
updated all this condition other places